src/util/helperUtil.ts

```diff
@@ -20,3 +22,20 @@ export function getConfiguration<T>(section: string, resourceOrFilepath?: vscode
 	}
 	return vscode.workspace.getConfiguration("liberty", resource).get<T>(section);
 }
+
+/**
+ * Filters the projects by command.
+ * @param projects The list of projects
+ * @param commamnd The command to check
+ * @returns a list of projects that the given command can be excuted on.
+ */
+export function filterProjects(projects: LibertyProject[], commamnd: string): LibertyProject[] {
+	let resultProjects: LibertyProject[] = [];
+	for ( const project of projects) {
+		const applicableTypes = COMMAND_AND_PROJECT_TYPE_MAP["liberty.dev.start"];
```

Comment:
Should "liberty.dev.start" here be replaced with the command?

Reply:
addressed